packages/language-server/src/svelte-check.ts

```diff
@@ -171,7 +171,7 @@ export class SvelteCheck {
                     // doesn't apply to all code paths. That's why we do it here, too.
                     const skipDiagnosticsForFile =
                         (options.skipLibCheck && file.isDeclarationFile) ||
-                        (options.skipDefaultLibCheck && file.hasNoDefaultLib ||
+                        (options.skipDefaultLibCheck && file.hasNoDefaultLib) ||
```

Comment:
Note - Non related change

Reply:
woops, I messed up when pushing to master,I just fixed that so you don't need that change anymore.

Reply:
👍   got it updated, thanks